packages/core/test/merkle-distributor/MerkleDistributor.js

```diff
@@ -251,7 +251,7 @@ contract("MerkleDistributor.js", function (accounts) {
           },
           { from: rando }
         );
-        assertApproximate(87749, claimTx.receipt.gasUsed);
+        assertApproximate(92049, claimTx.receipt.gasUsed);
```

Comment:
This is a bit odd -- why do you think these changed so much with a change in _hardhat_ version (not compiler version)?

Reply:
`hardhat@2.2` introduced support for `Berlin` as the default `hardfork`!